Sources/StreamChatUI/ChatMessageList/Attachments/VideoAttachmentCellView.swift

```diff
@@ -0,0 +1,120 @@
+//
+// Copyright © 2021 Stream.io Inc. All rights reserved.
+//
+
+import AVFoundation
+import StreamChat
+import UIKit
+
+/// The view used to display video attachment preview in a gallery inside a message cell
+typealias VideoAttachmentCellView = _VideoAttachmentCellView<NoExtraData>
+
+/// A view used to display video attachment preview in a gallery inside a message cell
+open class _VideoAttachmentCellView<ExtraData: ExtraDataTypes>: _View, ThemeProvider {
+    /// A video attachment the view displays
+    open var content: ChatMessageVideoAttachment? {
+        didSet { updateContentIfNeeded() }
+    }
+    
+    /// A handler that will be invoked when the view is tapped
+    open var didTapOnAttachment: ((ChatMessageVideoAttachment) -> Void)?
+    
+    /// An image view used to display video preview image
+    open private(set) lazy var imageView = UIImageView()
+        .withoutAutoresizingMaskConstraints
+
+    /// A loading indicator that is shown when preview is being loaded
+    open private(set) lazy var loadingIndicator = components
+        .loadingIndicator.init()
+        .withoutAutoresizingMaskConstraints
+
+    /// An uploading overlay that shows video uploading progress
+    open private(set) lazy var uploadingOverlay = components
+        .imageUploadingOverlay.init()
+        .withoutAutoresizingMaskConstraints
```

Comment:
Explicit type of these views are missing ⚠️

Reply:
✅ Provided